pkg/app/app.go

```diff
@@ -105,6 +105,18 @@ func GetFromSlug(slug string) (*App, error) {
 	return Get(id)
 }
 
+// UpdateDownstreamVersionStatus updates the status and status_info for the downstream version with the given sequence and app id
+func UpdateDownstreamVersionStatus(appID string, sequence int64, status string, statusInfo string) error {
```

Comment:
What's `statusInfo`? It appears to be an unused (static) parameter. Should we remove it?

Reply:
I tried to copy the same functionality from the typescript api. some functionalities there pass statusInfo to be updated. Figured i could add it here since we're gonna move to vendor-api in the near future.

Reply:
I don't really understand the comment about vendor-api? That's the internal api to ship a KOTS application, but this project never communicates with it, right?

I'm not a fan of adding parameters "for the future". I think that's how projects like this grow complex, i think we are better to leave future parameter definition and naming to the developers adding the work in the future?

Reply:
My bad, i meant kotsadm-api, and yes, that makes sense. I'll update.